### PR TITLE
Mention all-modules-page-plugin in README.md

### PR DESCRIPTION
**Pull request by martinbonnin**
I understand this is required for aggregation at the moment 

See https://github.com/adamko-dev/dokkatoo/issues/14

### CODE REVIEW

**Comment by aSemy** [comment]
thanks 🙏  this would help

File: README.md [hunk 1]
```diff
@@ -145,6 +145,9 @@ dependencies {
   // the subprojects must also have Dokkatoo applied
   dokkatoo(projects(":subproject-hello"))
   dokkatoo(projects(":subproject-world"))
+  
+  // This is required at the moment, see https://github.com/adamko-dev/dokkatoo/issues/14
+  dokkatooPluginHtml("org.jetbrains.dokka:all-modules-page-plugin:1.7.20")
```
Comment (aSemy) [inline README.md:150]: The version of Dokka that Dokkatoo uses by default is in a `Property`, which should be used so the version of Dokka is aligned.

```suggestion
  dokkatooPluginHtml(
    dokkatoo.versions.jetbrainsDokka.map { dokkaVersion ->
      "org.jetbrains.dokka:all-modules-page-plugin:$dokkaVersion"
    }
  )
```

But I haven't checked this - would you be able to?

Comment (martinbonnin) [reply README.md]: Alright, I think this works? 

```kotlin
dokkatooPluginHtml("org.jetbrains.dokka:all-modules-page-plugin:${org.jetbrains.dokka.DokkaVersion.version}")
```

Comment (aSemy) [reply README.md]: What about it doesn't work? Does the script compile, or do the Gradle tasks run but it doesn't produce an aggregated publication?

`dokkatoo` is the regular `DokkatooExtension` Gradle extension, `versions` is an extension added to `DokkatooExtension`, and `jetbrainsDokka` is a regular `Property<String>` on `DokkatooExtension.Versions`

Comment (martinbonnin) [reply README.md]: > What about it doesn't work? 

Problem between chair and computer 😅 . I forgot part of your snippet and was expecting a top level `dokkaVersion` property 🤦 . I'll update

Comment (martinbonnin) [reply README.md]: It's pretty hardcore to use without generated accessors though.

**Pull request merged**